src/_locales/en/messages.json

```
@@ -196,6 +196,18 @@
     "message": "Some settings are managed by your system administrator and cannot be changed.",
     "description": "The message, which appears, when settings are pre-defined (as managed options) by administrators."
   },
+  "titleGeneral": {
+    "message": "Add On",
+    "description": "The title for the general settings group."
+  },
+  "titleQrCodeSetting": {
+    "message": "QR Code Settings",
```

Possibly just like this?

```suggestion
    "message": "QR Code",
```

I mean, all options there are settings…

But if you have a different idea, feel free to comment.
Also, obviously, if you e.g. would re-order the options, that is totally possible. (e.g. monospace font is also only shown in the QR code popup, but mhh… I kinda like it this way and would also have no idea how to improve it.)